tarodb/assets_store.go

```diff
@@ -1600,6 +1605,43 @@ func (a *AssetStore) LogParcelDeliveryAttempt(ctx context.Context,
 	})
 }
 
+// QueryParcelDeliveryLog returns timestamps which correspond to logged parcel
+// delivery attempts.
+func (a *AssetStore) QueryParcelDeliveryLog(ctx context.Context,
+	anchorPoint wire.OutPoint) ([]time.Time, error) {
+
+	anchorPointBytes, err := encodeOutpoint(anchorPoint)
+	if err != nil {
+		return []time.Time{}, err
+	}
+
+	var timestamps []time.Time
+	readOpts := NewAssetStoreReadTx()
+
+	err = a.db.ExecTx(ctx, &readOpts, func(q ActiveAssetsStore) error {
+		// First, we'll fetch the asset transfer using the given
+		// outpoint bytes. We can then relate the delivery attempt to
+		// the asset transfer entry.
+		assetTransfers, err := q.QueryAssetTransfers(ctx, TransferQuery{
+			NewAnchorPoint: anchorPointBytes,
+		})
+		if err != nil {
+			return err
+		}
+		assetTransfer := assetTransfers[0]
+
+		timestamps, err = q.QueryReceiverProofTransferAttempt(
+			ctx, assetTransfer.TransferID,
+		)
+		if err != nil {
+			return err
+		}
+
+		return nil
+	})
+	return timestamps, err
```

Comment:
Same comment as Oli, maybe wrap this err to give it some context upper in the stack